packages/components/src/internal/util/WindowFocusCheckExpiredSession.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ComponentType, FC, useEffect } from 'react';
+
+import { useServerContext } from '../components/base/ServerContext';
+
+const useWindowFocusCheckExpiredSession = (): void => {
```

Comment:
nit: Consider also exporting `useWindowFocusCheckExpiredSession` so it can be used independent of `withWindowFocusCheckExpiredSession`.

Reply:
Yes, we can definitely do that. I didn't have a need for it on its own at the moment.